tasks/copy_to_glacier/schemas/output.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_files/schemas/output.json",
+  "title": "copy_to_files Lambda Output",
+  "description": "The output for the copy_to_files Lambda.",
+  "type": "object",
+  "required": ["granules", "copied_to_glacier"],
+  "properties": {
+    "granules": {
+      "description": "The 'granules' property from the input.",
+      "type": "array",
+      "items": {
+        "required": ["granuleId", "files"],
+        "properties": {
+          "granuleId": {
+            "description": "The granule's ID.",
+            "type": "string"
+          },
+          "files": {
+            "description": "The files to copy.",
+            "type": "array",
+            "items": {
+              "required": ["name", "bucket", "filepath", "filename"],
+              "properties": {
+                "name": {
+                  "type": "string"
```

Comment:
Should add a description to each of these properties so we know what they are and what to expect for a value.

Reply:
Gave it a shot.